clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureNetworkClient.groovy

```diff
@@ -572,7 +694,7 @@ class AzureNetworkClient extends AzureBaseClient {
    * @return the dns name of the given load balancer
    */
   String getDnsNameForLoadBalancer(String resourceGroupName, String loadBalancerName) {
-    String dnsName = "none"
+    String dnsName = "dns-not-found"
 
     try {
       def loadBalancer = executeOp({loadBalancerOps.get(resourceGroupName, loadBalancerName, null)})?.body
```

Comment:
Same as getting dns for app gateway. We've already read the item out of Azure. We shouldn't be reading it out again.